docs/workforce-management-framework/topics/pro-running-the-demo-repositories.adoc

```diff
@@ -68,7 +68,7 @@ NOTE: The terminal is out of use while the Core Server is running.
 
 [id='{context}-published-repositories-running-the-mobile-and-portal-applications']
 [discrete]
-== Running the Mobile and Portal {PROJECT_NAME} Applications
+== Running the Mobile and Portal Starter Applications
```

Comment:
After consultation with team, community etc. we decided on the names here.
We trying to stick with with `Reference Application` or just `Application` adding another option will be confusing. Starter seems off.

FYI @austincunningham @JameelB @witmicko